backend/schema/schema_test.go

```diff
@@ -55,19 +55,15 @@ func normaliseString(s string) string {
 func TestImports(t *testing.T) {
 	input := `
 	module test {
-		data Generic<T> {
-			value T
-		}
```

Comment:
Do we not want to test this still? How do we know if the imports are correct now?

Reply:
Ah whoops you're right! Got too enthusiastic with `git revert`!

Reply:
Restored that and added another test for verb references.